src/util/handleChangeDecimalNumber.js

```diff
@@ -11,24 +11,19 @@ import {includes, isEmpty, isNaN, last, size, split, toNumber} from 'lodash';
  * @param {Function} onChange Handle when the answer has changed.
  */
 const handleChangeDecimalNumber = ({name, allowZero, max, min}, value, onChange) => {
-    const splittedValue = split(value, '.');
     const parsedValue = toNumber(value);
+    const lastChar = last(value);
 
-    if (includes(value, '.') && size(splittedValue) === 2) {
-        return onChange({
-            [name]: isEmpty(last(splittedValue)) ? value : parsedValue
-        });
-    }
-    if (isEmpty(value) || isNaN(parsedValue)) {
-        return onChange({[name]: undefined});
+    if (lastChar === '.' || lastChar === ',') {
+        return null;
     }
     if (parsedValue < min) {
         return onChange({[name]: min});
     }
     if (parsedValue > max) {
         return onChange({[name]: max});
     }
-    if (parsedValue === 0 && !allowZero) {
+    if ((parsedValue === 0 && !allowZero) || value === '' || isNaN(parsedValue)) {
```

Comment:
Esta línea es similar a https://github.com/indec-it/react-native-form-builder/pull/69/files#diff-a083ae57281edd949fa4e5a74d501cadR3. No debería re-utilizar la función?

Reply:
Para le re utilización de esta función sería asi: 
```javascript
if (isEmptyAnswer(allowZero, value, parsedValue) || value === '') {
        return onChange({[name]: undefined});
}
``` 
o sino modificar el 'isEmptyAnswer', y dejarlos así:
```javascript
const isEmptyAnswer = (allowZero, answer, parsedAnswer) => (
    isNil(answer) || isNaN(parsedAnswer) || (parsedAnswer === 0 && !allowZero) || answer === ''
);
```
Nose que prefiere @navarroaxel ...

Reply:
la 2da está bien

Reply:
Listo